### PR TITLE
Adding propsTypes to styled components

### LINKED ISSUE #727: Add PropTypes for styled-components
As our styled-components get more complex, it might be beneficial to add PropType validation. In addition to catching bugs I think it would be a nice form of documentation.  Since everything is broken out into individual files now this makes more sense.

### PR DESCRIPTION
**Pull request by 7ruth**
Resolves #727

### CODE REVIEW

File: src/components/Main/components/Readout/components/ReadoutItem/components/ReadoutItemData.js [hunk 1]
```diff
@@ -15,4 +16,8 @@ const ReadoutItemData = styled.div`
   }
 `;
 
+ReadoutItemData.propTypes = {
+  paddingLeft: PropTypes.string
```
Comment (spmcbride1201) [inline src/components/Main/components/Readout/components/ReadoutItem/components/ReadoutItemData.js]: could be a number here I think

File: src/components/Main/components/TableCol.js [hunk 2]
```diff
@@ -10,4 +11,8 @@ const TableCol = styled.div`
     props.errorPercent ? `color: ${errorColor(props.errorPercent)}` : ""};
 `;
 
+TableCol.propTypes = {
+  errorPercent: PropTypes.string
```
Comment (spmcbride1201) [inline src/components/Main/components/TableCol.js]: could be a number

File: src/components/Main/components/Readout/components/ReadoutItem/components/ReadoutItemData.js (hunk 1 above)
Comment (7ruth) [reply src/components/Main/components/Readout/components/ReadoutItem/components/ReadoutItemData.js]: I checked our code base for this and I actually didn't see anywhere where paddingLeft prop is passed to readoutItemData...so I went with what I thought was logical. Adjusting to include both.

File: src/components/Main/components/TableCol.js (hunk 2 above)
Comment (7ruth) [reply src/components/Main/components/TableCol.js]: I checked 3 files where TableCol gets this prop (FunctionTable/../LineItem.js; RoutesTable/../LineItem.js; RoutesTableLineItem.js) and in all of those file the prop is a string, so unless you think that will change in the future I'd want to keep this as it is.

Comment (karinakeen) [reply src/components/Main/components/TableCol.js]: @7ruth They could be number, depending on how they get refactored for rendering..You could use PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.number
  ])

Comment (7ruth) [reply src/components/Main/components/TableCol.js]: Ok sounds like you expect it to change in the future... will adjust.

**Pull request merged**